src/cli/cli.cpp

```diff
@@ -4805,7 +4851,7 @@ template <> otError Interpreter::Process<Cmd("neighbor")>(Arg aArgs[])
          * Done
          * @endcode
          * @par
-         * Print connection time and age of neighbors.
+         * Prints the connection time and age of neighbors.
          * This command is similar to `neighbor conntime`, but it displays the information in a list format. The age
          * and connection time are both displayed in seconds.
```

Comment:
Can you add a `@csa` tag for the `neighbor conntime` command here?  So we have a link to it in the reference.

And vice versa, link conntime to conntime list